backend/app.coffee

```diff
@@ -49,6 +62,8 @@ class App
             if err
                 log.error err
                 log.error 'An error occured while registering your device.'
+                if url[0..6] is 'http://'
+                    log.warn 'Did you try with an httpS URL?'
```

Comment:
You have a `url` module with node that is proper to parse an url and analyze it.